src/OgAccess.php

```diff
@@ -186,18 +186,16 @@ public static function userAccessEntity($operation, EntityInterface $entity, Acc
     // @TODO: add caching on Og::isGroupContent.
     $is_group_content = Og::isGroupContent($entity_type_id, $bundle);
     $cache_tags = $entity_type->getListCacheTags();
-    if ($is_group_content && $entity_groups = Og::getEntityGroups($entity)) {
+
+    if ($is_group_content && $groups = Og::getGroups($entity)) {
```

Comment:
I need to add another check here that if the $entity is of type user but not a group, we should throw an exception.